cmd/juju/storage/import.go

```diff
@@ -128,6 +138,12 @@ func (c *importFilesystemCommand) Init(args []string) error {
 	return nil
 }
 
+// SetFlags implements Command.SetFlags.
+func (c *importFilesystemCommand) SetFlags(f *gnuflag.FlagSet) {
+	c.StorageCommandBase.SetFlags(f)
+	f.BoolVar(&c.force, "force", false, "Force import by deleting existing PVC if bound but not used by Juju (CAAS models only)")
```

Comment:
start with lower case, also we say "k8s" or "Kubernetes" not "CAAS" for user facing messages
we might not limit this to k8s models. there could be other scenarios.
we can make the message generic; the cli help text will explain use cases
eg "import a volume even if otherwise prohibited (cloud specific)"